manual/src/refman/typedecl.etex

```diff
@@ -196,10 +198,12 @@ constructor must have the same arity and the same type constraints as the
 original type constructor.
 \end{description}
 
-The type variables appearing as type parameters can optionally be
-prefixed by "+" or "-" to indicate that the type constructor is
-covariant or contravariant with respect to this parameter.  This
-variance information is used to decide subtyping relations when
+The type variables appearing as type parameters can optionally be prefixed by
+"+", "-", "+-", or "-+" to indicate that the type constructor has a specific
+variance with respect this parameter.
+A parameter with "+" is covariant, "-" contravariant,
+and "+-" (or "-+") bivariant.
```

Comment:
We can inline the definition of  bivariant here:
A parameter prefixed with "+" is covariant, one prefixed with "-" is contravariant.
A parameter can be both covariant and contravariant, also known as bivariant, with either
a "+-" or "-+" prefix.